### PR TITLE
Quarkus/OIDC update to app-profile-jee-vanilla issue 383

### PR DESCRIPTION
**Pull request by Toma5OD**
Changes made to ReadMe.md, pom.xml, Controller.java and web.xml.

File created app-profile-jee-vanilla/src/main/webapp/WEB-INF/oidc.json

Readme should be easily followed and understood. Application should be fully functional with Wildfly 27.

### CODE REVIEW

**Comment by pskopek** [comment]
@Toma5OD thanks for the PR. It needs to be placed against main branch (instead of the latest one).
You would like to reabase it as well as the last commit did change the pom.xml.

File: app-profile-jee-vanilla/README.md [hunk 1]
```diff
@@ -1,37 +1,25 @@
...
 Summary: JSP Profile Application with Basic Authentication  
 Target Product: <span>Keycloak</span>, <span>WildFly</span>  
-Source: <https://github.com/keycloak/keycloak-quickstarts>  
+Source: <https://github.com/keycloak/keycloak-quickstarts>
 
+### What is it?
 
-What is it?
------------
+The `app-profile-jee-vanilla` quickstart demonstrates how to change a JavaEE application that is secured with basic authentication without any changes to the WAR itself. Changing the authentication method and injecting the configuration is done automatically by the <span>Keycloak</span> client adapter subsystem.
```
Comment (pskopek) [inline app-profile-jee-vanilla/README.md:11]: The "JavaEE" should not be used here as we have transitioned to Jakarta EE.

Comment (pskopek) [inline app-profile-jee-vanilla/README.md:11]: The change of authentication method is not done automatically with Elytron OIDC adapter. This should be rephrased.

**Review by pskopek — CHANGES_REQUESTED** [review]

**Review by vmuzikar — COMMENTED** [review]
I believe this should target `main` instead of `latest`. Please see: https://github.com/keycloak/keycloak-quickstarts/blob/06db58ad5748c874f808419582382d8b829dcdfa/CONTRIBUTING.md?plain=1#L74-L78

**Pull request closed without merging**